qml/MonitorMenuBar.qml

```diff
@@ -87,6 +88,7 @@ MenuBar {
             onTriggered: {
                 inactive_visible = !inactive_visible
                 controller.change_inactive_visible()
+                editMenuRefresh.trigger()
```

Comment:
Please, move the logic to C++ when possible. Call refresh from the Engine and not from here

Reply:
Addressed in 97dc1c4